.travis.yml

```diff
@@ -5,14 +5,15 @@ sudo: false
 before_install:
   - unset GEM_PATH
 
-# ./gradlew assemble (default install command) won't cut it
 install:
-  - ./gradlew install
+  - ./gradlew install # ./gradlew assemble (default install command) won't cut it
+  - ./gradlew npm_install
```

Comment:
TODO: Break each linter into its own NPM script so they can be run independently